compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__hover__no_documentation_for_shared_record_field.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: compiler-core/src/language_server/tests/hover.rs
+expression: "\npub type Wibble {\n  Wibble(\n    /// This is some documentation about the wibble field.\n    wibble: Int\n  )\n  Wobble(\n    /// This won't show up because we don't know which wibble field it is\n    wibble: Int\n  )\n}\n\npub fn wibble(w: Wibble) {\n  w.wibble\n}\n"
+---
+pub type Wibble {
+  Wibble(
+    /// This is some documentation about the wibble field.
+    wibble: Int
+  )
+  Wobble(
+    /// This won't show up because we don't know which wibble field it is
+    wibble: Int
+  )
+}
+
+pub fn wibble(w: Wibble) {
+  w.wibble
+  ▔▔▔▔▔▔↑▔
+}
+
+
+----- Hover content -----
+Scalar(
+    String(
+        "```gleam\nInt\n```\n",
```

Comment:
Can we show the documentation for both in this instance? Seeing as extra docs is better than no docs